src/smtencoding/FStar.SMTEncoding.Solver.fs

```diff
@@ -220,20 +220,12 @@ let ask_and_report_errors env all_labels prefix query suffix =
             | ([], _), _
             | _, Inl _ -> result
             | _, Inr _ -> Inr errs in
+            
         let rec try_alt_configs prev_f (p:decl) (errs:z3_err) cfgs (scope:scope_t) =
             set_minimum_workable_fuel prev_f errs;
             match cfgs, snd errs with
             | [], _
             | _, Kill -> report p errs
-            | [mi], _-> //we're down to our last config; last ditch effort to get a counterexample with very low fuel
-                begin match errs with
-                | [], _ ->
-                    Z3.ask None all_labels (with_fuel [] p mi) (Some scope) (cb false mi p [] scope)
```

Comment:
I'm fine with removing min_fuel altogether. It is a bit hacky and I don't know of anyone really using the min_fuel option.